### DISCUSSION TITLE
The Arm Cloud Software Scavenger Hunt - October Winner

### DISCUSSION THREAD

**Discussion by zachlasiuk** [ArmDeveloperEcosystem/ecosystem-dashboard-for-arm#544] [Announcements]
## 👋 Welcome!
  This is where we place the results of the Arm Cloud Software Scavenger Hunt, highlighting individuals that contributed to the Arm ecosystem! 

  Those tagged below are recognized for their special contributions, and have won custom Arm SWAG as a result! Reply to this discussion thread to get a link to your reward :)

* Note: [Terms and Conditions for the Sweepstakes](https://www.arm.com/-/media/files/pdf/terms-and-conditions/arm-cloud-software-scavenger-hunt-terms-and-conditions)

**Comment by zachlasiuk** [comment]
@akshatnerella , you're our winner for October! Reply for instructions on how to get your SWAG :) Thanks for contributing the UV package to the dashboard: https://www.arm.com/developer-hub/ecosystem-dashboard/?package=uv

**Reply by akshatnerella** [reply]
thank you!

**Comment by akshatnerella** [comment]
Hello,
I got selected to receive ARM swag. Please let me know what's the next step.
Regards,
Akshat Nerella

**Reply by zachlasiuk** [reply]
Thanks again for contributing! I sent you the SWAG redemption link over at your LinkedIn profile to make sure you get it ASAP. Let me know once you've filled it out!

**Reply by akshatnerella** [reply]
I have filled it out thanks!
